From: Yaroslav Halchenko <debian@onerussian.com>
Subject: keep __init__.py pristine

Since packaging is based on GIT repository directly, createInitFile is present
in .orig.tar.gz, nevertheless we should not regenerate __init__.py while
setup.py operations are running since it might be ran not within repository any
longer.

--- a/setup.py
+++ b/setup.py
@@ -49,6 +49,7 @@ else:
 # regenerate __init__.py only if we're in the source repos (not in a source zip file)
 try:
     import createInitFile  # won't exist in a sdist.zip
+    raise Exception("createInitFile is shipped in Debian .orig.tar.gz but __init__.py should pristine")
     writeNewInit=True
 except:
     writeNewInit=False
